phoenicis-repository/src/main/java/org/phoenicis/repository/types/ClasspathRepository.java

```diff
@@ -296,14 +297,23 @@ private ScriptDTO buildScript(String typeId, String categoryId, String applicati
                 LOGGER.debug("Could not find script icon.");
             }
 
-            return new ScriptDTO.Builder(objectMapper.readValue(scriptJsonInputStream, ScriptDTO.class))
-                    .withTypeId(typeId)
-                    .withCategoryId(categoryId)
-                    .withApplicationId(applicationId)
-                    .withId(scriptFileName)
-                    .withScript(new String(IOUtils.toByteArray(scriptFile)))
-                    .withIcon(icon)
-                    .build();
+            ScriptDTO.Builder scriptDTOBuilder = new ScriptDTO.Builder(
+                    objectMapper.readValue(scriptJsonInputStream, ScriptDTO.class))
+                            .withTypeId(typeId)
+                            .withCategoryId(categoryId)
+                            .withApplicationId(applicationId)
+                            .withScript(new String(IOUtils.toByteArray(scriptFile)))
+                            .withIcon(icon);
```

Comment:
The indentation looks kind of strange here

Reply:
The formatter indents it like this. I cannot change it.